expression/builtin_string_vec.go

```diff
@@ -2868,3 +2866,91 @@ func (b *builtinCharLengthUTF8Sig) vecEvalInt(input *chunk.Chunk, result *chunk.
 	}
 	return nil
 }
+
+func formatDecimal(xBuf *chunk.Column, xDecimals []types.MyDecimal, dInt64s []int64, result *chunk.Column) error {
+	return formatDecimalWithLocale(nil, xBuf, xDecimals, dInt64s, result, nil)
+}
+
+func formatDecimalWithLocale(sctx sessionctx.Context, xBuf *chunk.Column, xDecimals []types.MyDecimal, dInt64s []int64, result *chunk.Column, localeBuf *chunk.Column) error {
```

Comment:
How about removing the argument `xDecimals` since it can be got by `xBuf.Decimals()`?